app/src/androidTest/java/org/mozilla/fenix/ui/MediaNotificationTest.kt

```diff
@@ -51,8 +51,10 @@ class MediaNotificationTest {
 
         navigationToolbar {
         }.enterURLAndEnterToBrowser(videoTestPage.url) {
+            verifyPageContent("Page content: video player")
             clickMediaPlayerPlayButton()
             waitForPlaybackToStart()
+            verifyPageContent("Media file is playing")
```

Comment:
waitForPlaybackToStart() verifies the same thing, maybe you can re-write that method if needed or remove it